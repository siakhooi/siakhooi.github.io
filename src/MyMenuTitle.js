import React from 'react'
import PropTypes from 'prop-types'

MyMenuTitle.propTypes = {
  title: PropTypes.string
}

function MyMenuTitle (props) {
  return (
    <span className="MyMenuTitle">{props.title}</span>
  )
}

export default MyMenuTitle
