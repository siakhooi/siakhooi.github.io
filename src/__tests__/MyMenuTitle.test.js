import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuTitle from '../MyMenuTitle'

it('MyMenuTitle/1', () => {
  const component =
    renderer.create(
      <MyMenuTitle
        title={'Data Science'}
      />
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
