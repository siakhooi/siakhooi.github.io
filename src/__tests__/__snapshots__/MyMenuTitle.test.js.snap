// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuTitle/1 1`] = `
<span
  className="MyMenuTitle"
>
  Data Science
</span>
`;
